pkg/k8s/kubernetes.go

```diff
@@ -637,11 +634,14 @@ func BuildSUT(client *kubernetes.Clientset, s *config.PerfScenarios) error {
 
 // Extract the UDN Ip address of the server (or the client) from the annotations - Support only ipv4
 func ExtractUdnIp(pod corev1.Pod) (string, error) {
+	for key, value := range pod.Annotations {
+		log.Debugf("Pod Annotation key: %s, value: %s\n", key, value)
```

Comment:
There's no need to add a new line with using a logging library.

```suggestion
		log.Debugf("Pod Annotation key: %s, value: %s", key, value)
```